frontend-instructor/src/components/coursesCreate/CreateContent.jsx

```
@@ -0,0 +1,367 @@
+
+    try {
+      let result;
+      if (item.video) {
+        result = await fetchData(checkVideo, { videoId: item.video.id });
+        setActiveForm('video');
+      } else if (item.quiz) {
+        result = await fetchData(getQuiz, { quizId: item.quiz.id });
+        setActiveForm('quiz');
+      }
+
+      if (result && result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)